README.md

```diff
@@ -419,6 +419,13 @@ These settings can be changed at any time.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
 * `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
 
+#### CloudFormation signal helper settings
+
+For AWS variants, these settings allows you to set up CloudFormation signalling to indicate whether Bottlerocket hosts running in EC2 have been successfully created or updated:
+* `settings.cloudformation.should-signal`: Whether to check status and send signal. Defaults to `false`.
+* `settings.cloudformation.stack-name`: Name of the CloudFormation Stack to signal.
+* `settings.cloudformation.logical-resource-id`: The logical ID of the AutoScalingGroup resource that you want to signal.
```

Comment:
Are both of these required? Will the instance will fail if `should-signal` is `true` and either of these is missing? Maybe document the behavior/dependency if there is one.

Reply:
default value for both of them is an empty string (`sources/models/shared-defaults/cf-signal.toml`), so you will get an error like (this is for python):
```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid length for parameter StackName, value: 0, valid min length: 1
```